### ISSUE TITLE
ember-config-service dependency

### ISSUE THREAD

**Issue by fusion2004** [sivakumar-kailasam/ember-purify#59]
It looks like in the 5.0 release, a dependency to the ember-config-service addon was introduced, but is not listed as a dependency in package.json.

I found it because our test suite started failing, saying it couldn't inject `service:config`, and looking at the commits I was able to figure it out.

It is listed as a devDependency for the dummy app, which is why the test suite here is still passing.

I'm admittedly not very knowledgable about addons or how they interact with each other, if we move `ember-config-service` from `devDependencies` to `dependencies`, will that fix it?

In the meantime, I've installed the addon in our app and that fixed the error for us.

**Comment by sivakumar-kailasam** [comment]
> I'm admittedly not very knowledgable about addons or how they interact with each other, if we move ember-config-service from devDependencies to dependencies, will that fix it?

Yes that should fix this issue. I've released a patch version with the fix.

Thanks for taking the time to report this 🙂

**Comment by fusion2004** [comment]
Thanks!
